src/components/Footer.js

```diff
@@ -164,17 +167,18 @@ class Footer extends React.Component {
                       textAlign={['center', null, 'left']}
                       mb={2}
                     >
-                      <StyledLink
-                        href={navigation[key][item]}
-                        color="#6E747A"
-                        display="block"
-                        fontSize="1.4rem"
-                        fontWeight="400"
-                        m={0}
-                        p={0}
-                      >
-                        {item}
-                      </StyledLink>
+                      <Link route={navigation[key][item]} passHref>
```

Comment:
Shouldn't this be instead within the definition of `<StyledLink />`? /cc @HipsterBrown

Reply:
It's what we're doing in other places, I think @HipsterBrown is good with that.